server/src/users/mod.rs

```diff
@@ -0,0 +1,11 @@
+use std::path::{Path, PathBuf};
+
+pub mod policy;
+
+const DEFAULT_USER_ID: &str = "default";
+
+// Fetch the users working directory.
+pub fn path(dir: &Path, id: Option<String>) -> PathBuf {
+    let id = id.unwrap_or_else(|| DEFAULT_USER_ID.to_string());
+    dir.join("users").join(id)
+}
```

Comment:
1. I see that in all the call-sites of this function we add another directory element:

```
let file = users::path(dir, id).join(tee) ...
```

maybe add the tee string as another argument to this function?

2. No sanitization/validation is done on value of `id` (or `tee`) which are received from the client request. They may contain `"../../../"` and not sure what this might cause.

Reply:
> maybe add the tee string as another argument to this function?

OK. I will add a new function which will includes the `tee` argument in `policy.rs`, as the following:
```
fn workdir(dir: &Path, id: Option<String>, tee: String) -> PathBuf {
    users::path(dir, id).join(tee)
}
```

> No sanitization/validation is done on value of id (or tee) which are received from the client request. They may contain "../../../" and not sure what this might cause

- The `tee` is restrict by `management_api::Tee` enum. So the only possible input is "sgx", "tdx", "sevsnp", and "sample".
- The `id` needs to be kept as `None` because the user id is not supported by guest client currently. Maybe we can add that checking when the user id is enabled in future.